base/document-download-api-deployment.yaml

```diff
@@ -51,7 +56,24 @@ spec:
               value: '$(NEW_RELIC_MONITOR_MODE)'
             - name: NOTIFY_ENVIRONMENT
               value: '$(ENVIRONMENT)'
-          resources: {}
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "400Mi"
+            limits:
+              cpu: "400m"
+              memory: "800Mi"
+          ports:
+            - containerPort: 7000
```

Comment:
Is not really needed https://github.com/kubernetes/kubernetes/issues/4332#issuecomment-448900396 but maps to the same values https://github.com/cds-snc/notification-manifests/blob/2772a8b4ea096c770538a7858b45ae01242425c7/base/document-download-api-service.yaml#L11-L12